tests/integration/kubernetes/k8s-confidential-attestation.bats

```diff
@@ -0,0 +1,67 @@
+#!/usr/bin/env bats
+# Copyright 2024 IBM Corporation
+# Copyright 2024 Intel Corporation
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+load "${BATS_TEST_DIRNAME}/../../common.bash"
+load "${BATS_TEST_DIRNAME}/confidential_common.sh"
+load "${BATS_TEST_DIRNAME}/tests_common.sh"
+
+export test_key="aatest"
+export SUPPORTED_TEE_HYPERVISORS=("qemu-sev" "qemu-snp" "qemu-tdx" "qemu-se")
+export SUPPORTED_NON_TEE_HYPERVISORS=("qemu")
+
+setup() {
+	if ! [[ " ${KATA_HYPERVISOR} " =~ " ${SUPPORTED_TEE_HYPERVISORS[@]} " ]] && ! [[ " ${KATA_HYPERVISOR} " =~ " ${SUPPORTED_NON_TEE_HYPERVISORS} " ]]; then
+		skip "Test not supported for ${KATA_HYPERVISOR}"
+	fi
+
+	get_pod_config_dir
+
+	# TODO need to get CC_KBS deployed with sample KBS based on info in
```

Comment:
@wainersm should I remove also this as seems like this will be implemented in https://github.com/kata-containers/kata-containers/blob/db33361e1347b729d868a121f75be0445fb3cef6/tests/integration/kubernetes/gha-run.sh#L101

Reply:
In this point it is assumed that KBS is installed and proper configured. However, it still needs to feed the KBS with test data.

We will need to set `test_key` to KBS. We will use the functions introduced in https://github.com/kata-containers/kata-containers/pull/9114):

```shell
kbs_set_resource "default" "aa" "key" "$test_key"
```

Then set the KBS policy to allow to release any resource unconditionally:
```shell
kbs_set_allow_all_resources
```

Reply:
@wainersm that is already added on https://github.com/kata-containers/kata-containers/pull/9137/files#diff-ff48cb2e0de04667da60ddff8920b131ce7b2ac8707899c3d660eb0a7d48653dR29